cypress/integration/e2e/main.js

```diff
@@ -0,0 +1,40 @@
+/// <reference types="cypress" />
+
+describe('E2E Tests', () => {
+    it('Search, open details and close it after assert', () => {
+        // Open site
+        cy.visit('http://127.0.0.1:3000')
```

Comment:
`http://127.0.0.1:3000` can be a constant to reuse it in test

Reply:
Done